docs/resiliency/index.asciidoc

```diff
@@ -296,7 +296,7 @@ the change, it is committed and commit messages are sent to the nodes. See {GIT}
 
 During master election each node pings in order to discover other nodes and validate the liveness of existing
 nodes. Based on this information the node either discovers an existing master or, if enough nodes are found
-(see https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery-zen.html#master-election[`discovery.zen.minimum_master_nodes`]) a new master will be elected. Currently, the node that is
+(`discovery.zen.minimum_master_nodes`) a new master will be elected. Currently, the node that is
 elected as master will update the cluster state to indicate the result of the election. Other nodes will submit
```

Comment:
Discovery has changed in 7.0 and the referenced setting is deprecated. Rather than simply removing the link, I'd remove the references to the discovery.zen.minimum_master_nodes setting and cross reference https://www.elastic.co/guide/en/elasticsearch/reference/7.0/modules-discovery-quorums.html#_master_elections for more information.

Reply:
Thanks @debadair! I removed those references with 2d4aef6.